invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -362,7 +385,8 @@ def get_related_identifiers(self, obj):
                 default=scheme,
             )
 
-            if id_scheme:
+            # Only serialize related identifiers with a valid scheme for DataCite.
+            if id_scheme and id_scheme.lower() in RELATED_IDENTIFIER_SCHEMES:
```

Comment:
minor: instead of that would it make sense to:

- remove the `"datacite": "Other"` value from the `RDM_RECORDS_IDENTIFIERS_SCHEMES` config (we only need "valid" DataCite values there)
- change it so that the `get_scheme_datacite` calls are a bit more strict? Right now I see `default=scheme` being passed in many places, which is wrong. We want to enforce that if there is not equivalent DataCite scheme for an identifier, it has to be skipped.

Reply:
I added a [task](https://github.com/inveniosoftware/invenio-rdm-records/issues/1584) to tackle the invalid schemes.

We have more schemes that don't map to DataCite's related identifiers (e.g. crossreffunderid)